tests/Unit/API/Trace/Propagation/TraceContextValidatorTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Tests\API\Unit\Trace;
+
+use OpenTelemetry\API\Trace\Propagation\TraceContextValidator;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @covers OpenTelemetry\API\Trace\TraceContextValidator
+ */
+class TraceContextValidatorTest extends TestCase
+{
+    private const INVALID_TRACE_FLAG = 'f';
+    private const TRACE_FLAG = '11';
+    private const TRACE_VERSION = '00';
+
+    public function test_is_invalid_trace_flag(): void
+    {
+        $this->assertFalse(TraceContextValidator::isValidTraceFlag(self::INVALID_TRACE_FLAG));
+    }
+
+    public function test_is_valid_trace_flag(): void
+    {
+        $this->assertTrue(TraceContextValidator::isValidTraceFlag(self::TRACE_FLAG));
+    }
+
+    public function test_is_invalid_trace_version(): void
```

Comment:
Change the function name to test_valid_trace_version as '00' is a valid trace version and the test asserts that. Also, add another test to test_invalid_trace_version and add 'ff' to that as it is an invalid version according to [this](https://www.w3.org/TR/trace-context/#version).

Reply:
done